docs/sources/helm-charts/mimir-distributed/get-started-helm-charts/_index.md

```diff
@@ -136,48 +136,6 @@ Using a custom namespace solves problems later on because you do not have to ove
 
 1. Wait until all of the pods have a status of `Running` or `Completed`, which might take a few minutes.
 
-## Generate some metrics for testing
```

Comment:
@eamonryan Just confirming that we want to remove this entire section.

Reply:
Yes, I don't think it's good to include since it uses the deprecated Operator mode.